model/Image.php

```diff
@@ -240,77 +243,87 @@ public function SetRatioSize($width, $height) {
 		}
 
 		// Item must be regenerated
-		return  $this->getFormattedImage('SetRatioSize', $width, $height);
+		return  $this->getFormattedImage('Fit', $width, $height);
 	}
 
 	/**
-	 * Resize the image by preserving aspect ratio, keeping the image inside the
-	 * $width and $height
+	 * Scale image proportionally to fit within the specified bounds
 	 *
 	 * @param Image_Backend $backend
 	 * @param integer $width The width to size within
 	 * @param integer $height The height to size within
 	 * @return Image_Backend
 	 */
-	public function generateSetRatioSize(Image_Backend $backend, $width, $height) {
```

Comment:
Oh sorry. I was thinking it isn't relevant anymore but it's still part of the API isn't it. Noted.